proto/cosmos/gov/v1/gov.proto

```diff
@@ -81,6 +81,14 @@ message Proposal {
 
   // metadata is any arbitrary metadata attached to the proposal.
   string metadata = 10;
+
+  // title is the title of the proposal
+  string title = 11;
+
+  // summary is a short summary of the proposal
+  string summary = 12;
+
+  string proposer = 13 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
Odd...I swear proposer was already part of the proposal. I thought I reviewed a migration PR to add this.

Reply:
It's part of `MsgSubmitProposal` already. I guess block explorers use the original tx msg to display it. That said, do we really need to store it?

Reply:
I was thinking the same, not sure if it's worth it.

Reply:
afair #13010 is already adding it. Should have been a separate PR but @gsk967 preferred to add it in single PR

Reply:
ah okay, ill add it here then, Ill complete this pr today